packages/angular/ssr/src/app-engine.ts

```diff
@@ -124,7 +124,7 @@ export class AngularAppEngine {
       return undefined;
     }
 
-    const entryPointExports = await entryPoint();
+    const entryPointExports = entryPoint();
```

Comment:
NIT: the `async` is not redundant in `getEntryPointExports`.

Reply:
Good point